### ISSUE TITLE
[EPIC] Port "Tags" bulk update to bulk action

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/argo#2007]
Part of #1914 

Port the updating of tags from a bulk update to a bulk action

- [ ] Unrevert the [reversion PR](https://github.com/sul-dlss/argo/pull/2053) (note `NOTE`s in code)
- [ ] Update the related infrastructure integration test. (see [draft PR](https://github.com/sul-dlss/infrastructure-integration-test/pull/96) as a start)

**Comment by andrewjbtw** [comment]
A few notes on current behavior in the bulk updates (this came up recently in dlss-aaas Slack):

- there are two boxes to take input for this bulk update
    1. A box that takes the list of druids to update
    2. A box that takes a tab-separated list of druids + tags

The second box behaves differently depending on how you get the list of druids:

- populate from search: the box of druids + tags auto-populates the druids plust current tags from the search
- paste in a druid list: the box of druids + tags does not auto-populate

Users have requested that the box auto-populate when pasting in a druid list. See https://github.com/sul-dlss/argo/issues/741 

Alternatively, we could provide some other mechanism for retrieving existing tags for an arbitrary set of druids. If users could update multiple hitherto-unrelated druids at once this would greatly help them. 

A prime use case for bulk tag updates is adding a tag to many objects that, prior to adding the tag the user is trying to add, have not been grouped together.

**Comment by andrewjbtw** [comment]
Another bit of current behavior that seems important: Argo overwrites the current tags with whatever is supplied in the bulk tag update. This seems to be why it's necessary to retrieve the current tags before making edits.

One way around retrieving current tags would be to provide bulk "add a tag" functionality where the tag is added to the current tags rather than replacing them.

**Comment by mjgiarlo** [comment]
@aaron-collier @andrewjbtw I think we ought to meet soon to discuss the requirements for this work.

**Comment by mjgiarlo** [comment]
@aaron-collier @andrewjbtw we should discuss this to unblock it. Moved to backlog.

**Comment by mjgiarlo** [comment]
Discussed with @aaron-collier and @andrewjbtw and we came up with the following:

We will use #2007 as an epic for replacing the tags bulk update with new bulk actions. It will wrap the following tickets:

1. Add new "append tag(s)" bulk action
1. Add new "remove tag(s)" bulk action
1. Add new "export tags" bulk action
1. Add new "import tags" bulk action
1. Decommission bulk update tag (unrevert #2053)

**Comment by andrewjbtw** [comment]
I think we can consider this complete as we have import/export tags functionality in the bulk actions. We may finally want to retire the old way to update tags but that can be a separate ticket.
